ansible_base/authentication/management/commands/authenticators.py

```diff
@@ -73,9 +73,9 @@ def initialize_authenticators(self):
                 create_objects=True,
                 configuration={},
                 created_by=admin_user,
-                created_on=now(),
```

Comment:
Why `created` and `modified` are passed explicitly here? Wouldn't it make sense to remove it?

Reply:
Irrelevant to this PR:

```
self.stderr.write("Error message")
exit(255)
```

can be replaced with the `CommandError` exception, provided by Django

```
raise CommandError("Error message")
```

Reply:
Good catch, I've updated this. r.e. the CommandError I filed a new issue https://github.com/ansible/django-ansible-base/issues/211.